### PR TITLE
Apps-Service: Add baptiste-preview to ListRepos and ListUserRepos

### PR DESCRIPTION
**Pull request by andygrunwald**
The preview `application/vnd.github.baptiste-preview+json` ensures that the field `"is_template": false` is part of the repository response of the Apps Service.

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @andygrunwald !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
💙

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
